spec/features/creating_a_drcf_digital_markets_research_spec.rb

```diff
@@ -0,0 +1,153 @@
+require "spec_helper"
+
+RSpec.feature "Creating a DRCF digital markets research", type: :feature do
+  let(:drcf_digital_markets_research) { FactoryBot.create(:drcf_digital_markets_research) }
+  let(:content_id) { drcf_digital_markets_research["content_id"] }
+  let(:save_button_disable_with_message) { page.find_button("Save as draft")["data-disable-with"] }
+
+  before do
+    log_in_as_editor(:drcf_digital_markets_research_editor)
+    allow(SecureRandom).to receive(:uuid).and_return(content_id)
+
+    stub_publishing_api_has_content([drcf_digital_markets_research], hash_including(document_type: DrcfDigitalMarketsResearch.document_type))
+    stub_publishing_api_has_item(drcf_digital_markets_research)
+    stub_any_publishing_api_put_content
+    stub_any_publishing_api_patch_links
+  end
+
+  scenario "visiting the new research page" do
+    visit "/drcf-digital-markets-researches"
+    click_link "Add another DRCF digital markets research"
+    expect(page.status_code).to eq(200)
+    expect(page.current_path).to eq("/drcf-digital-markets-researches/new")
+  end
+
+  scenario "creating the new DRCF digital markets research with no data" do
+    visit "/drcf-digital-markets-researches/new"
+
+    click_button "Save as draft"
+
+    expect(page.status_code).to eq(422)
+
+    expect(page).to have_css(".elements-error-summary")
+    expect(page).to have_css(".form-group.elements-error")
+    expect(page).to have_css(".elements-error-message")
```

Comment:
I can see this is in some of the other tests but not too sure of the value of it, they just tell us some html elements exists which isn't particularly of value to a user - would suggest removing these lines.